docs/src/test/scala/docs/http/scaladsl/HttpClientExampleSpec.scala

```diff
@@ -71,6 +71,50 @@ class HttpClientExampleSpec extends WordSpec with Matchers with CompileOnlySpec
     //#manual-entity-consume-example-2
   }
 
+  "manual-entity-consume-example-3" in compileOnlySpec {
+    //#manual-entity-consume-example-3
+    import scala.concurrent.duration._
+    import scala.concurrent.Future
+
+    import akka.NotUsed
+    import akka.actor.ActorSystem
+    import akka.http.scaladsl.Http
+    import akka.http.scaladsl.model._
+    import akka.stream.ActorMaterializer
+    import akka.util.ByteString
+    import akka.stream.scaladsl.{Flow, Sink, Source}
```

Comment:
This is wrongly formatted and broke the build. This should fix it
```suggestion
    import akka.stream.scaladsl.{ Flow, Sink, Source }
```

Reply:
Build is failing because of this line. Please note the missing spaces after `{` and before `}`